scripts/src/models/users.ts

```diff
@@ -23,8 +30,44 @@ export class User extends CreationDateModel {
 	@OneToMany(type => OrderContext, context => context.user)
 	public contexts!: OrderContext[];
 
-	@Column({ name: "wallet_count", default: 1 })
+	@Column({ name: "wallet_count", default: User.defaultWalletCount })
 	public walletCount!: number;
+
+	/**
+	 * Overrided save method
+	 * If this (user) is new, it calls direct insert method instead of built-in upsert TypeORM functionality
+	 * It generates id and tries to insert it to the table, up to 3 tries, and breaks the loops on success
+	 */
+	public async save(): Promise<this> {
+		if (!this.isNew) { await super.save(); return this; }
+
+		let errorCount = 0;
+		const triesCount = 3;
+		while (true) {
+			if (errorCount > triesCount) { throw new Error(`user generated with the same id more than ${ triesCount } times or some another error`); }
+
+			try { // tries to insert a new user with generated id
+				await getManager()
+					.createQueryBuilder()
+					.insert()
+					.into(User)
+					.values([{
```

Comment:
the problem with this is that we'll need to remember that each field needs to be specified here once added to the class, so it's error-prone.
from the docs/definition files/examples it seems like you can just do:
`.values([this])`
and if you use the `@Initializer` decoration (as my comment above suggested) then you also won't have a null in the `this.walletCount` member.